### ISSUE TITLE
Error when calling PyHTCC with Euro sign in password

### ISSUE THREAD

**Issue by SamueldeVries** [csm10495/pyhtcc#16]
Hi! 
Thanks for sharing the repository. I came across the following bug: 

When using € sign in a password calling PyHTCC gives:
UnicodeEncodeError: 'latin-1' codec can't encode character '\u20ac' in position X: ordinal not in range(256)

Where X is the character position of the euro sign in the password.

**Comment by csm10495** [comment]
Hey,

Mind checking your hosts locale/encoding?

See https://stackoverflow.com/a/57298412/3824093

I think if you set it to utf8, this would go away. If not, can you give more of the traceback?

**Comment by csm10495** [comment]
Ah that's not going to be it. Seems like this is broken at the requests level.. I'll see if i can handle it.

**Comment by csm10495** [comment]
Now this looks better: 
```
PS C:\Users\csm10495\Desktop\pyhtcc> python -c "from pyhtcc import PyHTCC; PyHTCC('user', '?')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\csm10495\Desktop\pyhtcc\pyhtcc\pyhtcc.py", line 478, in __init__
    self.authenticate()
  File "C:\Users\csm10495\Desktop\pyhtcc\pyhtcc\pyhtcc.py", line 491, in authenticate
    return self._do_authenticate()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\csm10495\Desktop\pyhtcc\pyhtcc\pyhtcc.py", line 550, in _do_authenticate
    raise LoginCredentialsInvalidError(
pyhtcc.pyhtcc.LoginCredentialsInvalidError: Email (user) and/or password appear to have been rejected
```
.. release inbound.

**Comment by csm10495** [comment]
I released 0.1.56 that should fix this. Thanks for reporting!
